sips/sip-007/sip-007-stacking-consensus.md

```diff
@@ -517,16 +517,18 @@ the second through nth outputs:
 1. If the block commitment is in a reward cycle, with a chosen anchor block, and this block
    commitment builds off a descendant of the PoX anchor block (or the anchor block itself),
    then the commitment must use the chosen PoX recipients for the current block.
-    a. PoX recipients are chosen as described in "Stacking Consensus Algorithm": addresses
-       are chosen without replacement, by using the previous burn block's sortition hash,
-       mixed with the previous burn block's burn header hash as the seed for the ChaCha12
-       pseudorandom function to select M addresses.
-    b. The leader block commit transaction must use the selected M addresses as outputs [1, M]
-       That is, the second through (M+1)th output correspond to the select PoX addresses.
-       The order of these addresses does not matter. Each of these outputs must receive the
-       same amount of BTC.
-    c. If the number of remaining addresses in the reward set N is less than M, then the leader
-       block commit transaction must burn BTC by including (M-N) burn outputs.
+    <ol type="a">
```

Comment:
Hey, can we use markdown here instead of HTML? Thanks

Reply:
Sure, I'll change it.